JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -163,10 +162,10 @@ private fun Paragraph(paragraph: Paragraph) {
 
     val annotatedString = paragraphToAnnotatedString(
         paragraph,
-        MaterialTheme.typography(),
-        MaterialTheme.colors().codeBlockBackground
+        MaterialTheme.typography,
+        MaterialTheme.colors.codeBlockBackground
     )
-    Container(modifier = LayoutPadding(0.dp, 0.dp, 0.dp, bottom = trailingPadding)) {
+    Box(modifier = Modifier.padding(0.dp, 0.dp, 0.dp, bottom = trailingPadding)) {
```

Comment:
nit: if using named params I tend to prefer using it for all params.

Reply:
Good catch, this is very confusing.